### DISCUSSION TITLE
🐓 v3.11.x

### DISCUSSION THREAD

**Discussion by trexfeathers** [SciTools/iris#5571] [Releases]
<div align="center">

| Manager | Buddy | Milestone | Feature Branch | Project |
| :---: | :---: | :---: | :---: | :---: |
| @ESadek-MO | @HGWright | [v3.11](https://github.com/SciTools/iris/milestone/51) | [v3.11.x](https://github.com/SciTools/iris/tree/v3.11.x) | [v3.11 Project](https://github.com/orgs/SciTools/projects/37/views/1?pane=issue&itemId=79109297&issue=SciTools%7Ciris%7C6043) |

</div>

<div align="center">

| Minor Release | Status | Date | RTD | Release | conda-forge | PyPI |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| `v3.11.0rc0` | [![status](https://img.shields.io/badge/v3.11.0rc0-released-green)](https://github.com/SciTools/iris/discussions/5571) | `28-10-2024` | [RTD](https://scitools-iris.readthedocs.io/en/v3.11.0rc0/) | [Release](https://github.com/SciTools/iris/releases/tag/v3.11.0rc0) | [conda-forge](https://anaconda.org/conda-forge/iris?version=3.11.0rc0)  | [PyPi](https://pypi.org/project/scitools-iris/3.11.0rc0/#files) |
| `v3.11.0` | [![status](https://img.shields.io/badge/v3.11.0-released-green)](https://github.com/SciTools/iris/discussions/5571) | `11-11-2024` | [RTD](https://scitools-iris.readthedocs.io/en/v3.11.0/) | [Release](https://github.com/SciTools/iris/releases/tag/v3.11.0) | [conda-forge](https://anaconda.org/conda-forge/iris?version=3.11.0) | [PyPI](https://pypi.org/project/scitools-iris/3.11.0/) |
| `v3.11.1` | [![status](https://img.shields.io/badge/v3.11.1-released-green)](https://github.com/SciTools/iris/discussions/5571) | `19-12-2024` | [RTD](https://scitools-iris.readthedocs.io/en/v3.11.1/) | [Release](https://github.com/SciTools/iris/releases/tag/v3.11.1) | [conda-forge](https://anaconda.org/conda-forge/iris?version=3.11.1) | [PyPI](https://pypi.org/project/scitools-iris/3.11.1/) |

</div>

**Comment by bjlittle** [comment]
This is probably the earliest `numpy>=2.0` version of `iris` that we can aim for.

Of course, this all depends on the compliance of our package dependencies ...

**Reply by trexfeathers** [reply]
- https://github.com/SciTools/iris/pull/6035

**Reply by trexfeathers** [reply]
Note that @cgsandford has kindly offered to trial the NumPy v2 compliant version of Iris against her use cases, so we should get in touch when the release candidate is near.

**Comment by valeriupredoi** [comment]
following this since we are just about there to support numpy >=2 (pending an approval from reviewer) in ESMValCore https://github.com/ESMValGroup/ESMValCore/pull/2395 - can help here too, if more hands on deck are needed :beer:

**Comment by bouweandela** [comment]
It would be really nice this bugfix could be included in the release: https://github.com/SciTools/iris/pull/6187

**Comment by ESadek-MO** [comment]
Release candidate has been released!

**Reply by ESadek-MO** [reply]
P.S. apologies if you picked up the release candidate from the main conda-forge channel this afternoon. This oversight is being fixed as we speak. You can check this with:

`conda list -n my_env | grep iris`

**Comment by valeriupredoi** [comment]
just pointing folks to this - good news on a Monday :beer: https://github.com/ESMValGroup/ESMValCore/pull/2566#issuecomment-2450193985

**Comment by ESadek-MO** [comment]
The release is out!

**Reply by valeriupredoi** [reply]
and I have already tested with it in ESMValCore (with numpy>2) - cheers, guys! :beer:

**Comment by ESadek-MO** [comment]
v3.11.1 is out, with a pin for `dask` and a `cfunits` bug fix!
